devicetypes/ezex/somfy-blind-controller-ezex.src/somfy-blind-controller-ezex.groovy

```diff
@@ -0,0 +1,120 @@
+/**
+ *
+ *	Copyright 2018 eZEX
+ *
+ *	Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ *	in compliance with the License. You may obtain a copy of the License at:
+ *
+ *		http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *	Unless required by applicable law or agreed to in writing, software distributed under the License is distributed
+ *	on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License
+ *	for the specific language governing permissions and limitations under the License.
+ */
+ 
+metadata {
+    definition(name: "SOMFY Blind Controller/eZEX", namespace: "eZEX", author: "eZEX Corp", vid:"SmartThings-smartthings-Somfy_WindowShade", ocfDeviceType: "oic.d.blind") {
+        capability "Actuator"
+        capability "Configuration"
+        capability "Refresh"
+        capability "Window Shade"
+        capability "Health Check"
+        capability "Switch Level"
+        
+        command "pause"
+
+        fingerprint profileId: "0104", inClusters: "0000, 0003, 0004, 0102", outClusters: "0019", deviceJoinName: "SOMFY Blind Controller/eZEX", model: "E2B0-KR000Z0-HA"// SY-IoT201-BD
```

Comment:
By convention the `deviceJoinName` should be the last thing in the fingerprint. Is `SOMFY Blind Controller/eZEX` what you want to show up when the device joins?

Reply:
It was from the partner I didn't check it. Is the deviceJoinName to override to name in the definition for each FP?
Then in this case, can I just change the sequence like manufacturer:  , model: and delete the deviceJoinName?

Reply:
If you want to do join name translations you'll need to keep the join name on the fingerprint but otherwise it's not technically needed and the DTH name will be used as the join name. You do need to include the `model` (and in general `manufacturer` if the device supports it) since that's what is really used for identifying the device.

Reply:
I will apply like that.